pkg/webhooks/workload_webhook_test.go

```diff
@@ -232,6 +234,35 @@ func TestValidateWorkload(t *testing.T) {
 				field.Invalid(firstPodSetSpecPath.Child("containers").Index(0).Child("resources", "requests").Key(string(corev1.ResourcePods)), nil, ""),
 			},
 		},
+		"should have a valid podSetUpdate name": {
```

Comment:
Please also add tests which verify the set of names in `podSetUpdates` to be not matching these in `podSets`

Reply:
added https://github.com/kubernetes-sigs/kueue/pull/1220/commits/0943f326b3dc45a32e58f6899a8bbf4b755e07db